src/WebJobs.Script.WebHost/WebJobsScriptHostService.cs

```diff
@@ -222,6 +222,12 @@ private void CheckFileSystem()
         /// </summary>
         private async Task UnsynchronizedStartHostAsync(ScriptHostStartupOperation activeOperation, int attemptCount = 0, JobHostStartupMode startupMode = JobHostStartupMode.Normal)
         {
+            CheckFileSystem();
```

Comment:
Chose to centralize this, rather than add a CheckFileSystem call to RestartHostAsync which also would have worked.